protocol-units/settlement/mcr/contracts/src/settlement/MCR.sol

```diff
@@ -22,6 +29,26 @@ contract MCR is Initializable, BaseSettlement, MCRStorage, IMCR {
         leadingBlockTolerance = _leadingBlockTolerance;
         lastAcceptedBlockHeight = _lastAcceptedBlockHeight;
         stakingContract.registerDomain(_epochDuration, _custodians);
+        addCommitmentAdmin(msg.sender);
+        addTrustedAttester(msg.sender);
+    }
+
+    function addCommitmentAdmin(address account) public {
```

Comment:
not necessary, DEFAULT_ADMIN_ROLE is everyone's admin, can add anyone with grantRole